tests/xarray/test_clone_using.py

```diff
@@ -0,0 +1,65 @@
+import xarray as xr
+import numpy as np
+import pytest
+
+import janitor  # noqa: F401
+
+
+@pytest.mark.xarray
+def test_successful_cloning_coords(da):
+
+    # with copying coords
+    new_da: xr.DataArray = da.clone_using(np.random.randn(*da.data.shape))
```

Comment:
I've never seen this syntax before! Can you educate me on this?

Reply:
The `new_da: xr.` bit? That’s an in-line type hint for mypy because I like the autocomplete and it wasn’t happening with the monkey patching taking place haha.

Reply:
Oh cool, it works in-line too? Wow noice :smile:

Reply:
Glorious I know